assets/javascript/app.js

```diff
@@ -218,4 +223,4 @@ $(".dropdown-item").on("click", function() {
 $("#theaterSubmit").on("click", function(event) {
   event.preventDefault();
   $("#food-form").modal("show");
-});
\ No newline at end of file
+});
```

Comment:
Ignore these changes at the end of the file, my text editor does that, I think its a mac/windows thing.